internal/server/db/db.go

```diff
@@ -16,17 +17,22 @@ func (d *Database) New(comp compute.Compute) {
 }
```

Comment:
Чтобы избежать работы с неинициализированными объектами лучше использовать конструктор
```suggestion
func NewDatabase(comp compute.Compute) *Database {
    return &Database{comp: comp}
}

```